src/pkg/logger/logger.go

```diff
@@ -64,27 +74,28 @@ const (
 // flag values
 var (
 	DebugAPIFV          bool
-	logFileFV           = ""
-	LogFormatFV         = string(LFText)
-	LogLevelFV          = string(LLInfo)
+	logFileFV           string
+	LogFormatFV         string
+	LogLevelFV          string
 	ReadableLogsFV      bool
 	MaskSensitiveDataFV bool
 
-	LogFile string // logFileFV after processing
+	LogFile     string // logFileFV after processing
+	PIIHandling string // piiHandling after MaskSensitiveDataFV processing
```

Comment:
do these need to be exported? Are they used?

Reply:
guess not.